src/api/XRWebGLLayer.js

```diff
@@ -74,26 +75,11 @@ export default class XRWebGLLayer extends XRLayer {
   get antialias() { return this[PRIVATE].config.antialias; }
 
   /**
-   * @return {boolean}
-   */
-  get depth() { return this[PRIVATE].config.depth; }
-
-  /**
-   * @return {boolean}
-   */
-  get stencil() { return this[PRIVATE].config.stencil; }
-
-  /**
-   * @return {boolean}
-   */
-  get alpha() { return this[PRIVATE].config.alpha; }
```

Comment:
These were actually removed from the spec, and it was an area where since I was messing around with the file anyway I figured I'd just go ahead and make the signature match.